tests/tasks/Cloudwatch/cw-metric.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name:  cloudwatch
+  namespace: tekton-pipelines
+spec:
+  description: "cw task to push metrics relying on generate task."
```

Comment:
We should talk about what it does in the description. e.g. `cw task push one data point to AWS CloudWatch`.